Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/RelationshipReactivationRequestTests.cs

```diff
@@ -7,30 +7,30 @@
 using static Backbone.Modules.Relationships.Domain.Tests.TestHelpers.TestData;
 
 namespace Backbone.Modules.Relationships.Domain.Tests.Tests.Aggregates.Relationships;
-public class ReactivateRelationshipRequestTests
+public class RelationshipReactivationRequestTests
```

Comment:
A test is missing that asserts that reactivation can only be requested if the relationship is in status "Terminated".

Reply:
Test already exists `Cannot_Reactivate_An_Already_Requested_Reactivation`.